duckduckgo/iOS
# 15. Subscription sign-out notification observers and Design Feedback

Task/Issue URL: https://app.asana.com/0/414235014887631/1206458412095677/f

**Description**:
- Design review feedback updates (copy, layout fixes, etc)
- Removes debug flags on alpha builds (cc @graeme)
- Add notification observers for signOut on notifications



**Steps to test this PR**:
1.  Validate The subscription restore page looks fine, compared to this reference:
<img width="433" alt="Screenshot 2024-03-01 at 3 48 16 PM" src="https://github.com/duckduckgo/iOS/assets/1156669/ad1978c3-2569-4351-978e-98084e1886c3">

2.  Whenever possible, leave settings page open, put app in background and come back after subscription is expired (Should refresh settings)
